### ISSUE TITLE
Add links to docker/openSUSE registry and MS Store

### ISSUE THREAD

**Issue by hellcp** [openSUSE/get-o-o#4]
We have some representation in those places, it would be good to link to that from here

**Comment by himanshu007-creator** [comment]
HI, i would like to work on this issue

**Comment by hellcp** [comment]
Sure, do you need some help?

**Comment by himanshu007-creator** [comment]
> Sure, do you need some help?

Yes, where do I have to insert links. Can you point out the file/lines?
Thanks

**Comment by hellcp** [comment]
Probably somewhere in `/_data/{151,152,153,tumbleweed,kubic,microos}.yml`, and adjust `/_includes/distribution.html` to display them correctly

**Comment by akshat5302** [comment]
hey, I want to contribute to this issue is this issue still in open mode ??

**Comment by hellcp** [comment]
Yes

**Comment by akshat5302** [comment]
Sorry @hellcp I just didn't read your comment I would like to contribute to this issue

**Comment by akshat5302** [comment]
From the above comments, I have figured out that I have to insert links but could you just tell me what are the links and from where I could find them. @hellcp

**Comment by hellcp** [comment]
That's probably a question to a release manager like @lkocman, I'm not sure which links are official there.

**Comment by akshat5302** [comment]
Ohk should I tag them here or you will ask him in personal

**Comment by akshat5302** [comment]
Hey, @hellcp Have u asked them about what links to add??

**Comment by hellcp** [comment]
I haven't

**Comment by akshat5302** [comment]
Should I ask him here ??

**Comment by hellcp** [comment]
I don't know
